src/main.cpp

```diff
@@ -55,6 +80,10 @@ int main(int argc, char **argv)
     LauncherLocaleManager *launcherLocaleManager = new LauncherLocaleManager();
     QObject::connect(app.localeManager(), SIGNAL(localeChanged()), launcherLocaleManager, SLOT(onLocaleChanged()));
 
+    // Create the app launcher and expose it to QML
+    AppLauncher *appLauncher = new AppLauncher();
```

Comment:
This style is not consistent with the rest of the codebase. Use `qmlRegisterType<AppLauncher>("org.asteroid.launcher", 1, 0, "AppLauncher");`